reconcile/utils/gql.py

```diff
@@ -177,6 +177,10 @@ def init(url, token=None, integration=None, validate_schemas=False):
     return _gqlapi
 
 
+def get_resource(path):
```

Comment:
I know this is a very simple function, but without types it's still going to be assumed that the return is `Any`. As such, could we add types here?

Reply:
sure steve [a839526](https://github.com/app-sre/qontract-reconcile/pull/2303/commits/a839526175d688d296ed4dcc0b53160eb4506f2c)